docs/content/docs/getting-started/generate-providers.md

```diff
@@ -32,6 +33,10 @@ to reduce the generation time.
 make terraform VERSION=ga OUTPUT_PATH="$GOPATH/src/github.com/hashicorp/terraform-provider-google" PRODUCT=foo
 ```
 
+## Build command guide
+
+{{< build_table >}}
```

Comment:
Please remove the documentation in the section above for PRODUCT (since it duplicates this table) and add documentation to the table for RESOURCE.

Reply:
I'm not sure what best to do with the example of a nonsense product - maybe the more complex examples could go in a separate "examples" section after the command reference?

Reply:
ack, refactored the page, let me know what you think